auth/pkg/user_v1/user_grpc.pb.go

```diff
@@ -0,0 +1,220 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
This file should not be here. It's supposed that it will be generated by other developer using `protoc`